azure-storage-blob/info.json

```diff
@@ -0,0 +1,110 @@
+                "required": true,
+                "editable": true,
+                "visible": true
+            },
+            {
+                "title": "Verify SSL",
+                "required": false,
+                "editable": true,
+                "visible": true,
+                "type": "checkbox",
+                "name": "verify_ssl",
+                "value": true,
+                "description": "Specifies whether the SSL certificate for the server is to be verified or not.\nBy default, this option is set to True."
+            },
+            {
+                "title": "Container Name",
+                "required": true,
+                "editable": true,
+                "visible": true,
+                "type": "text",
+                "name": "container_name",
+                "description": "Specify the name of the azure container."
+            }
+        ]
+    },
+    "operations": [
+        {
+            "title": "Put Blob",
+            "description": "Creates a new blob or replaces an existing blob within a container.",
+            "operation": "create_blob",
+            "category": "investigation",
+            "annotation": "create_blob",
+            "parameters": [
+                {
+                    "title": "Blob Name",
+                    "required": true,
+                    "editable": true,
+                    "visible": true,
+                    "type": "text",
+                    "name": "blob_name",
+                    "tooltip": "Specify the name of the blob to create or replace.",
+                    "description": "Specify the name of the blob to create or replace."
+                },
+                {
+                    "title": "Blob Type",
+                    "required": true,
+                    "editable": true,
+                    "visible": true,
+                    "type": "select",
+                    "name": "blob_type",
+                    "options": [
+                      "AppendBlob",
+                      "PageBlob",
+                      "BlockBlob"
+                    ],
+                    "tooltip": "Specify the type of the blob to create block blob, page blob, append blob.",
+                    "description": "Specify the name of the blob to create or replace."
+                },
+                {
+                    "title": "Blob Data",
+                    "required": false,
+                    "editable": true,
+                    "visible": true,
+                    "type": "text",
+                    "name": "blob_data",
+                    "value": "",
+                    "tooltip": "Specify the content of the blob. For a page blob or an append blob, the content is empty.",
+                    "description": "Specify the name of the blob to create or replace."
+                }
+            ]
+        },
+        {
+            "title": "List Blob",
```

Comment:
Action doesn't have an output schema

Reply:
This REST API returns only xml response. Please let me know how can I add a xml response in output schema.

Reply:
We generally convert the xml response into json. so you would have to convert it and copy the json in output schema so that it can be used further.